lib/Runtime/Library/JavascriptExternalFunction.cpp

```diff
@@ -273,7 +273,13 @@ namespace Js
 
         ScriptContext * scriptContext = externalFunction->type->GetScriptContext();
         AnalysisAssert(scriptContext);
-        Var result = NULL;
+        Var result = nullptr;
+
+        StdCallJavascriptMethodInfo info = {
+            callInfo.Count > 0 ? args[0] : scriptContext->GetLibrary()->GetNull(),
```

Comment:
I believe that we should always have a `this` parameter, so `args[0]` should always be passed; I'd prefer an assert that `callInfo.count > 0` and just putting `args[0]` here rather than null (which isn't possible in normal JS scenarios)

Reply:
I'm onboard with this.